mlflow/store/db/utils.py

```diff
@@ -158,3 +162,18 @@ def _upgrade_db_initialized_before_mlflow_1(url):
     # add metric steps, do not need to depend on this one. This allows us to eventually remove this
     # method and the associated migration e.g. in MLflow 1.1.
     command.stamp(config, "base")
+
+
+def create_sqlalchemy_engine(db_uri):
+    pool_size = os.environ.get(MLFLOW_SQLALCHEMYSTORE_POOL_SIZE, "")
```

Comment:
nit: I think `os.environ.get(MLFLOW_SQLALCHEMYSTORE_POOL_SIZE)` would have the same behavior since it returns `None`.

Reply:
Done